synapse/event_auth.py

```diff
@@ -80,6 +80,7 @@ def validate_event_for_room_version(event: "EventBase") -> None:
     Raises:
         SynapseError if there is a problem with the event
     """
+
```

Comment:
nitty nit: there's a stray newline change in the overall diff here
and the same in `_check_power_levels` down below.

Reply:
I think I got them, let me know if it doesn't look right

Reply:
They're both gone, thank you!